app/samples/datalogger/native_sim.overlay

```diff
@@ -0,0 +1,81 @@
+/ {
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+
+		storage = &flash0;
+		logfs = &lfs1;
+	};
+
+
+	fstab {
+		compatible = "zephyr,fstab";
+		lfs1: lfs1 {
+			compatible = "zephyr,fstab,littlefs";
+			mount-point = "/lfs";
+			partition = <&external_storage_partition>;
+			automount;
+
+			// the binding reccomends defaults for these
+			read-size = <16>;
+			prog-size = <16>;
+			cache-size = <64>; // may need to grow for optimization
+			lookahead-size = <32>;
+			block-cycles = <(90 * 1000)>;
+			// the datasheet gives 100K P-E cycles
+			// NOTE: this seems to default to 512 for the mounted filesystem
+			// anyway
+		};
+	};
```

Comment:
Should look into making these snippets one day

Reply:
true. until then i will copy and paste the code yev wrote 12 months ago lol